backend/lib/endpoints/schema/posts.js

```diff
@@ -3,7 +3,9 @@ const S = require("fluent-schema");
 const getPostsSchema = {
   querystring: S.object()
     .prop("authorId", S.string())
+    // Number of items to skip
```

Comment:
I don't think there should be comments here, I'm  more in favor of the "clean code" principles to strive for self-documenting code and only add comments when absolutely needed

Reply:
Good point. Will remove the comment.